networkx/algorithms/flow/tests/test_maxflow.py

```diff
@@ -100,7 +100,7 @@ def test_graph1(self):
 
     def test_graph2(self):
         # A more complex undirected graph
-        # adapted from www.topcoder.com/tc?module=Statc&d1=tutorials&d2=maxFlow
```

Comment:
> Error
Invalid request
You may click [here](javascript:history.back();) to return to the last page you were viewing.
If you have a question or comment, please email [service@topcoder.com](mailto:service@topcoder.com) and be sure to include this timestamp: Sun Jun 18 15:50:37 EDT 2023.

Reply:
https://www.topcoder.com/thrive/articles/Maximum%20Flow:%20Part%20One the new one.

Let's future proof this https://web.archive.org/web/20220815055650/https://www.topcoder.com/thrive/articles/Maximum%20Flow:%20Part%20One